style.css

```diff
@@ -4,17 +4,121 @@
 }
 
 body {
-  display: block;
-  margin: 10px;
-  background-color: #fff;
-  border-radius: 0 0 0 5%;
+  background-color: #e5e5e5;
+  margin: 0;
+  font-family: Poppins, sans-serif;
 }
 
 footer {
   display: flex;
   flex-direction: column;
 }
 
+a {
+  text-decoration: none;
+}
+
+li {
+  list-style: none;
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 20px;
+  background-color: teal;
+}
+
+.nav-links a {
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #344563;
+}
```

Comment:
As requested by the previous reviewer, kindly make sure that your `nav` element doesn't occupy the whole width 😄:
| Figma design  | Your design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/42799579/201269126-6adbed89-cfe6-4c4d-86d2-d32f7e7dd682.png)  | ![image](https://user-images.githubusercontent.com/42799579/201269076-bf99ebb3-0aed-4845-a03f-ad144fb93583.png)  |

Check the space between the elements and the body border in the original design 😉.

> Note: you can always click on the images to see them with more details.